manifest.json

```diff
@@ -1723,12 +1723,12 @@
       "version": "1.2.2"
     },
     {
-      "description": "Provides official NodeJs binaries",
+      "description": "Provides official NodeJs binaries.",
       "id": "nodejs",
       "mod_version": "3",
-      "remote": "https://github.com/lite-xl/lite-xl-lsp-servers:89efc5642cb9abcd9023c8e5ef3dba85b399884e",
+      "remote": "https://github.com/lite-xl/lite-xl-lsp-servers:0cf6a6598826508187349fd5a79d77441b171e22",
```

Comment:
0cf6a6598826508187349fd5a79d77441b171e22 doesn't exist in that repo.
You need to wait until your other PR is merged to have the correct commit hash.

Reply:
Thanks